sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -382,3 +382,27 @@ function buildRawAuthorizationRule(authorizationRule: AuthorizationRule): any {
   };
   return rawAuthorizationRule;
 }
+
+/**
+ * @ignore
+ * Recurse through a JSON structure and remove key: value pairs where the value == `undefined`.
```

Comment:
Just to be clear, is the intent of this method is to remove `undefined` and `null` fields, not just `undefined`? If so, the method name is misleading/docs could be updated to call this out.

Reply:
Yes, updated docs